build/pom.xml

```diff
@@ -462,6 +462,11 @@
                     <artifactId>zbox</artifactId>
                     <version>${project.version}</version>
                 </dependency>
+                <dependency>
```

Comment:
You moved the dependency to the "premium" profile - that is not expected.

Reply:
There is no org.zstack:log4j2 module under zstack and the maven can not find this dependency during building. So i guess the module is under premium version and move it there. Any suggestion about the placement of org.zstack:log4j2 dependency?

Reply:
@live4thee Could we just remove this log4j2 dependency?

Reply:
I checked the code base and it turned out that your patch is correct.  I will merge it.  Thanks.

Reply:
Merged.